hw/ip/tlul/doc/dv_plan/index.md

```diff
@@ -108,8 +108,7 @@ We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/RE
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
 Here's how to run a basic sanity test:
 ```console
-$ cd hw/ip/xbar/dv
-$ make TEST_NAME=xbar_sanity
+$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim_cfg.hjson -i xbar_main_sanity
```

Comment:
@weicaiyang there are `xbar_main` and `xbar_peri`. I only add xbar_main here but i feel like maybe we need some description about how to run xbar_main and xbar_peri here.

Reply:
I dont think we should add chip specific stuff to this file. Lets discuss more on this offline.